src/main/kotlin/Menu.kt

```diff
@@ -0,0 +1,30 @@
+class Menu {
+    private val menuFormatter = "%d. %s" // формат вывода
+    private val menuItems = mutableListOf<MenuItem>()
+
+    fun render(list: List<File>, fileType: String) {
+        menuItems.clear()
+        menuItems.add(MenuItem("Создать $fileType"))
+        menuItems.addAll(list.map { MenuItem(it.name) })
+        menuItems.add(MenuItem("Выход"))
+
+        menuItems.forEachIndexed { index, archiveMenuItem ->
+            println(String.format(menuFormatter, index, archiveMenuItem.title))
```

Comment:
🍏 В Kotlin есть удобный механизм формирования строк - templates.
https://kotlinlang.org/docs/strings.html#string-templates

```suggestion
            println("$index. ${archiveMenuItem. title}")
```